components/eamxx/src/atmosphere_driver.hpp

```diff
@@ -0,0 +1,264 @@
+#ifndef SCREAM_ATMOSPHERE_DRIVER_HPP
```

Comment:
I'm assuming you did not mean to create these copies of the AD in the src folder?